### ISSUE TITLE
Paths are interpreted incorrectly on windows

### ISSUE THREAD

**Issue by StarpTech** [apparatus/fuge-runner#16]
Hi,
im trying the example and run into following issue. The `yamlPath` and `path` are interpreted incorrectly during the process. The slashes will be removed. Bewlow you can see the output of the `container.specific` on line https://github.com/apparatus/fuge-runner/blob/master/lib/processRunner.js#L57

Steps to rerpoduce

1. fuge generate system
2. fuge shell
3. start all

```
? fuge> start all
running: frontend
{ yamlPath: 'C:UsersdustinDesktopdustin\fuge/docker-compose.yml',
  source:
   { build: '../site/',
     container_name: 'site',
     env_file: './env',
     ports: [ '10000:20000' ] },
  path: 'C:UsersdustinDesktopdustin\fuge/../site/',
  type: 'process',
  proxyPort: 10000,
  servicePort: 20000,
  buildScript: 'echo NO BUILD SCRIPT!',
  execute:
   { exec: 'node index.js',
     environment: [],
     process: 'node index.js' },
  environment:
   { PROXY_HOST: '__TARGETIP__',
     SERVICE_HOST: '0.0.0.0',
     frontend_PORT: 10000,
     api_PORT: 10001,
     service1_PORT: 10002,
     service2_PORT: 10003,
     SERVICE_PORT: 20000 } }
running: api
{ yamlPath: 'C:UsersdustinDesktopdustin\fuge/docker-compose.yml',
  source:
   { build: '../api/',
     container_name: 'api',
     env_file: './env',
     ports: [ '10001:20001' ] },
  path: 'C:UsersdustinDesktopdustin\fuge/../api/',
  type: 'process',
  proxyPort: 10001,
  servicePort: 20001,
  buildScript: 'echo NO BUILD SCRIPT!',
  execute:
   { exec: 'node index.js',
     environment: [],
     process: 'node index.js' },
  environment:
   { PROXY_HOST: '__TARGETIP__',
     SERVICE_HOST: '0.0.0.0',
     frontend_PORT: 10000,
     api_PORT: 10001,
     service1_PORT: 10002,
     service2_PORT: 10003,
     SERVICE_PORT: 20001 } }
running: service1
{ yamlPath: 'C:UsersdustinDesktopdustin\fuge/docker-compose.yml',
  source:
   { build: '../service1/',
     container_name: 'service1',
     env_file: './env',
     ports: [ '10002:20002' ] },
  path: 'C:UsersdustinDesktopdustin\fuge/../service1/',
  type: 'process',
  proxyPort: 10002,
  servicePort: 20002,
  buildScript: 'echo NO BUILD SCRIPT!',
  execute:
   { exec: 'node service.js',
     environment: [],
     process: 'node service.js' },
  environment:
   { PROXY_HOST: '__TARGETIP__',
     SERVICE_HOST: '0.0.0.0',
     frontend_PORT: 10000,
     api_PORT: 10001,
     service1_PORT: 10002,
     service2_PORT: 10003,
     SERVICE_PORT: 20002 } }
running: service2
{ yamlPath: 'C:UsersdustinDesktopdustin\fuge/docker-compose.yml',
  source:
   { build: '../service2/',
     container_name: 'service2',
     env_file: './env',
     ports: [ '10003:20003' ] },
  path: 'C:UsersdustinDesktopdustin\fuge/../service2/',
  type: 'process',
  proxyPort: 10003,
  servicePort: 20003,
  buildScript: 'echo NO BUILD SCRIPT!',
  execute:
   { exec: 'node service.js',
     environment: [],
     process: 'node service.js' },
  environment:
   { PROXY_HOST: '__TARGETIP__',
     SERVICE_HOST: '0.0.0.0',
     frontend_PORT: 10000,
     api_PORT: 10001,
     service1_PORT: 10002,
     service2_PORT: 10003,
     SERVICE_PORT: 20003 } }
running: __proxy
```

The correct path is `'C:Users\dustin\Desktop\dustin\fuge/docker-compose.yml',`

Thank you. If it works on windows it would be awesome.

**Comment by mcdonnelldean** [comment]
I see other path issues in this too. Self assigned. cc @mcollina @lucamaraschi

**Comment by mcdonnelldean** [comment]
Seem feedback here,

The issue is to do with executing the generated command on Windows, we need to strip and re add quotes correctly. Issue has been reported in Node generally but we will add a work around here too.

**Comment by mcdonnelldean** [comment]
Fixed via #18
